src/datamigration/azext_datamigration/manual/helper.py

```diff
@@ -27,6 +29,24 @@
 from azure.cli.core.azclierror import InvalidArgumentValueError
 
 
+# -----------------------------------------------------------------------------------------------------------------
+# Common helper function to check if string is empty or not
+# -----------------------------------------------------------------------------------------------------------------
+def is_string_null_or_empty(value):
+    return value is None or value == "" or value.strip() == ""
+
+
+# -----------------------------------------------------------------------------------------------------------------
+# Helper function to check if an input string has escape characters.
+# -----------------------------------------------------------------------------------------------------------------
+def has_shell_escape_characters(inputString):
+    escape_characters = ['&', '|', ';', '<', '>', '(', ')', '[', ']', '{', '}', '$', '`', '\\', '"', "'", ' ', '\t', '\n', '\r']
+    for character in escape_characters:
```

Comment:
Nit : https://stackoverflow.com/questions/14411633/fastest-way-to-check-if-a-string-contains-specific-characters-in-any-of-the-item
Probably can make it faster

Reply:
Done. Added the one line of python.